scripts/pre_setups/init_lava_only_with_node.sh

```diff
@@ -51,13 +51,18 @@ screen -d -m -S provider1 bash -c "source ~/.bashrc; lavap rpcprovider \
 $PROVIDER1_LISTENER LAV1 rest '$LAVA_REST' \
 $PROVIDER1_LISTENER LAV1 tendermintrpc '$LAVA_RPC,$LAVA_RPC_WS' \
 $PROVIDER1_LISTENER LAV1 grpc '$LAVA_GRPC' \
-$EXTRA_PROVIDER_FLAGS --geolocation 1 --log_level trace --from servicer1 --chain-id lava --metrics-listen-address ":7776" 2>&1 | tee $LOGS_DIR/PROVIDER1.log" && sleep 0.25
+$EXTRA_PROVIDER_FLAGS --geolocation 1 --log_level trace --from servicer1 --chain-id lava --relay-concurrent-load-limit 1 --metrics-listen-address ":7776" 2>&1 | tee $LOGS_DIR/PROVIDER1.log" && sleep 0.25
```

Comment:
later on if we add a rate limit that impacts consumer side we wont know what went wrong if we forget to remove the 1 limit. lets remove it from the script.

Reply:
Fixed in [8b8a05a](https://github.com/lavanet/lava/pull/1720/commits/8b8a05acd54c009763e0aa8de852218165b398f4)